.github/workflows/formatter.yml

```diff
@@ -0,0 +1,11 @@
+on: push
+name: Code Format Checker
+jobs:
+  lint:
+    name: Code Format Checker
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checker
+      uses: kumagi/clang-format-action@check-diff
```

Comment:
Can you include the version of clang-format in the tag?

Reply:
Done.